src/test/java/org/opensearch/security/dlic/rest/api/AccountApiActionConfigValidationsTest.java

```diff
@@ -56,13 +54,13 @@ public void updatePassword() {
         requestContent.put("password", "cccccc");
         result = accountApiAction.updatePassword(SecurityConfiguration.of(requestContent, "u", configuration));
         assertTrue(result.isValid());
-        assertTrue(OpenBSDBCrypt.checkPassword(u.getHash(), "cccccc".toCharArray()));
+        assertTrue(passwordHasher.check("cccccc".toCharArray(), u.getHash()));
 
         requestContent.remove("password");
-        requestContent.put("hash", Utils.hash("dddddd".toCharArray()));
+        requestContent.put("hash", passwordHasher.hash("dddddd".toCharArray()));
         result = accountApiAction.updatePassword(SecurityConfiguration.of(requestContent, "u", configuration));
         assertTrue(result.isValid());
-        assertTrue(OpenBSDBCrypt.checkPassword(u.getHash(), "dddddd".toCharArray()));
+        assertTrue(passwordHasher.check("dddddd".toCharArray(), u.getHash()));
```

Comment:
AFAIK we started to use `hamcrest`. So lets use `assertThat(...)` from it instead